src/main/kotlin/no/nav/familie/ks/sak/kjerne/beregning/BeregningService.kt

```diff
@@ -77,7 +77,7 @@ class BeregningService(
             andelerTilkjentYtelseOgEndreteUtbetalingerService.finnEndreteUtbetalingerMedAndelerTilkjentYtelse(behandling.id)
                 .filter {
                     when {
-                        endretUtbetalingAndel != null -> it.id == endretUtbetalingAndel.id
+                        endretUtbetalingAndel != null -> it.id == endretUtbetalingAndel.id || it.andelerTilkjentYtelse.isNotEmpty()
```

Comment:
filter-betingelsen kunne nå kanskje vært forenklet litt?
`.filter { it.id == endretUtbetalingAndel?.id || it.andelerTilkjentYtelse.isNotEmpty() }`